Formula/qemu.rb

```diff
@@ -62,7 +63,8 @@ def install
     end
 
     system "./configure", *args
-    system "make", "V=1", "install"
+    system "ninja", "-v"
```

Comment:
This hunk is invalid. There's a dependency on ninja but it shouldn't be used directly yet.